src/irmin-pack/conf.mli

```diff
@@ -39,6 +39,14 @@ module type S = sig
         information). *)
 
   val inode_child_order : inode_child_order
+
+  val forbid_empty_dir_persistence : bool
+  (** If [true], irmin-pack raises [Failure] if it is asked to save the empty
+      inode. This default is [false]. It should be set to [true] if the [Schema]
```

Comment:
```suggestion
      inode. The default is [false]. It should be set to [true] if the [Schema]
```
Maybe the default should be true? In this way, we don't forget to set it true when we update `lib_context`

Reply:
As we've fixed the default functors, this can now be allowed by default.

We will have a type error when updating lib_context because of the missing `forbid_empty_dir_persistence` value in the conf, I think we are safe with the `false` default